src/App.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Routes, Route, useLocation, Navigate,
+} from 'react-router-dom';
+import './Style/register.css';
+import './Style/experienceDetails.css';
+import './Style/medias.css';
+import { loadUserFromLocalStorage } from './Redux/Slices/usersSlice';
+import LoginPage from './Components/LoginPage';
+import Register from './Components/Register';
+import RoutesWrapper from './Components/RoutesWrapper';
+import Navbar from './Components/NavBar';
+
+function App() {
```

Comment:
- [ ] You've done a great job so far, but I couldn't find a way to add or delete experiences.  These are core [requirements ](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md) for a team of 4 people.

Kindly, add these 2 missing features in order to improve your chances in the final evaluation. 💪 


_note:  According to this:_

![image](https://github.com/HFG43/final_capstone_front_end/assets/57421823/d8429ed8-d575-4582-965c-e0cd3a47332f)

If you had a delay due to some unexpected issues, kindly include an explanation in your PR description and we will take it into account.

Reply:
@Meltrust Thank you so much for all your feedback! We have been working on the project based on all your comments! Very much appreciated!